app/models/review.rb

```diff
@@ -1,16 +1,20 @@
+# model/review.rb
 class Review < ApplicationRecord
-
   # user will review many movies
-	belongs_to :user
-	# post will have many reviews
-	belongs_to :movie
-	validates :user_id, presence: { message: 'Your review must belong to a user'}
-	validates :movie_id, presence: { message: 'Your review must belong to a movie'}
-  validates :rating, presence: { message: 'You must give the movie a rating'}
-
-	# the driver (owner of the post) can review multiple passengers, but not the same passenger in the same post
+  belongs_to :user
+  # movie will have many reviews
+  belongs_to :movie
+  validates :user_id, presence: { message: 'Your review must belong to a user' }
+  validates :movie_id, presence: { message: 'Your review must belong to a movie' }
+  validates :rating, presence: { message: 'You must give the movie a rating' }
+  validates_numericality_of :rating, only_integer: true, allow_nil: false,
+                                     greater_than_or_equal_to: 1,
+                                     less_than_or_equal_to: 5,
+                                     message: 'Rating can only be whole number between 1 and 5.'
```

Comment:
I love giving 3.675 ratings :(

Reply:
I really debated whether to allow users to give specific number ratings. However, after reading about it online, I decided that they should only be able to "like" (rating of 5) or "dislike" (rating of 1) a movie. However if you can make a case for allowing specific ratings, then I'll change it.